epub34/annotations/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
+	<head>
+		<meta charset="utf-8" />
+		<meta name="color-scheme" content="light dark" />
+		<title>EPUB Annotations 3.4</title>
+		<script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove"></script>
+		<script src="../../common/js/css-inline.js" class="remove"></script>
+		<script class="remove">
+			var respecConfig = {
+				group: "pm",
+				wgPublicList: "public-pm-wg",
+				specStatus: "ED",
+				shortName: "epub-ann",
+				noRecTrack: false,
+				edDraftURI: "https://w3c.github.io/epub-specs/wg-notes/annotations/",
```

Comment:
I'm not sure about this URL, which contains "wg-notes".

Reply:
Ya, this is what must have confused me about the title. I was sure I'd seen it as a wg-note file...